scriptmodules/supplementary/retroscraper.sh

```diff
@@ -0,0 +1,410 @@
+        params+=" --disk"
+    fi
+
+    if [[ "$version" -eq 1 ]]; then
+        params+=" --version"
+    fi
+
+    if [[ "$hack" -eq 1 ]]; then
+        params+=" --hack"
+    fi
+
+    if [[ "$brackets" -eq 1 ]]; then
+        params+=" --brackets"
+    fi
+
+    if [[ "$bezels" -eq 1 ]]; then
+        params+=" --bezels"
+    fi
+
+    if [[ "$sysbezels" -eq 1 ]]; then
+        params+=" --sysbezels"
+    fi
+
+    if [[ "$cleanmedia" -eq 1 ]]; then
+        params+=" --cleanmedia"
+    fi
+
+    echo "$params"
+
+    # trap ctrl+c and return if pressed (rather than exiting retropie-setup etc)
+    trap 'trap 2; return 1' INT
+    #echo "su -c  python3 -u $md_inst/retroscraper.py ${params[@]} $user 2>&1 | dialog --backtitle "$__backtitle" --progressbox Scraping 22 76" > /tmp/test.txt
+    local cmd=(dialog --backtitle "$__backtitle" --prgbox "Scraping roms with RetroScraper" "su $user -c  'python3 $md_inst/retroscraper.py ${params[@]}'" 22 76)
+    local choice=$("${cmd[@]}" 2>&1 >/dev/tty)
+    trap 2
+}
+
+function scrape_chosen_retroscraper() {
+    local options=()
+    local system
+    local i=1
+    while read system; do
+        system=${system/$romdir\//}
+        options+=($i "$system" OFF)
+        ((i++))
+    done < <(list_systems_retroscraper)
+
+    local cmd=(dialog --backtitle "$__backtitle" --checklist "Select Systems" 22 76 16)
+    local choice=($("${cmd[@]}" "${options[@]}" 2>&1 >/dev/tty))
+
+    [[ ${#choice[@]} -eq 0 ]] && return
+
+    local choices
+    for choice in "${choice[@]}"; do
+        choices+="${options[choice*3-2]},"
+    done
+    scrape_retroscraper "$choices" "$@"
+}
+
+function select_lang_retroscraper(){
+    local options=()
+    local language
+    local i=1
+    local lan
+    while IFS=',' read -r short desc
+    do
+        options+=("$short" "$desc" OFF)
+    done < <(list_langs_retroscraper)
+
+    local cmd=(dialog --backtitle "$__backtitle" --radiolist "Select Language" 22 76 16)
+    local choice=($("${cmd[@]}" "${options[@]}" 2>&1 >/dev/tty))
+
+    [[ ${#choice[@]} -eq 0 ]] && return
+
+    #local choices
+    #for choice in "${choice[@]}"; do
+    #    choices="${options[choice*3-2]},"
+    #done
+    echo "$choice"
+}
+
+function _load_config_retroscraper() {
+    echo "$(loadModuleConfig \
+        'use_thumbs=1' \
+        'screenshots=0' \
+        'max_width=400' \
+        'max_height=400' \
+        'console_src=1' \
+        'mame_src=2' \
+        'rom_name=0' \
+        'append_only=0' \
+        'use_rom_folder=0' \
+        'download_videos=0' \
+        'download_marquees=0' \
+    )"
+}
+
+function gui_retroscraper() {
+    if pgrep "emulationstatio" >/dev/null; then
+        printMsgs "dialog" "This scraper must not be run while Emulation Station is running or the scraped data will be overwritten. \n\nPlease quit from Emulation Station, and run RetroPie-Setup from the terminal"
+        return
+    fi
+
+    iniConfig " = " '"' "$configdir/all/scraper.cfg"
```

Comment:
This config file is already used by the [scraper](https://github.com/RetroPie/RetroPie-Setup/blob/master/scriptmodules/supplementary/scraper.sh) module. You should probably use your own file (`retroscraper.cfg`) instead of clobbering that one.